deploy/operator.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: helm-operator
```

Comment:
this name sounds very generic. will this conflict with other helm-operators that the clusters may have? Also, can we include the standard openebs component labeling details here. On a related note, how is the helm-operator be versioned? is it going to follow the same as openebs versions?

Reply:
@kmova , updated the operator deployment to `openebs-helm-operator`. This deployment (Along with the RBAC, CRD) is for manual installation - i.e., anyone who wants to use the helm-operator in a standalone way. In case of OLM, the name used in `openebs-operator` (https://github.com/openebs/helm-operator/blob/bb4028aa812d9ea34856d5c89c5626091b90520f/olm/openebsoperator.v1.0.0.clusterserviceversion.yaml#L309)

For versioning of the helm-operator itself - version metadata is maintained in the CSV itself: https://github.com/openebs/helm-operator/blob/bb4028aa812d9ea34856d5c89c5626091b90520f/olm/openebsoperator.v1.0.0.clusterserviceversion.yaml#L204

General practice, as per other operators is to trail the main release - so we can follow same as openebs versions.